internal/osbuild/ostree_input.go

```diff
@@ -20,7 +20,7 @@ type OSTreeCheckoutInput struct {
 	References OSTreeCheckoutReferences `json:"references"`
 }
 
-func (OSTreeCheckoutInput) isStageInput() {}
+func (OSTreeCheckoutInput) isInput() {}
```

Comment:
From what I checked, this interface https://github.com/osbuild/osbuild-composer/blob/a56c70d5053b1c2e5293d68dc3665eefe694829f/internal/osbuild/input.go#L8-L11 is not really used by any code, there are only two instances of its use in the ostree stage / input implementations. This is IMO useless and I propose to instead delete it (probably all of the interface "implementations" as well as the interface). This is most probably a result of the initial inputs implementation when we didn't have everything figured out.

Reply:
Ack.